rosdep/python.yaml

```diff
@@ -2859,6 +2859,9 @@ python-pyaudio:
   fedora: [pyaudio]
   gentoo: [dev-python/pyaudio]
   ubuntu: [python-pyaudio]
+python-pycurl:
+  debian: [python-pycurl]
+  ubuntu: [python-pycurl]
```

Comment:
This is also available in Fedora as https://apps.fedoraproject.org/packages/python2-pycurl

Reply:
added in 7b03eba